src/_guides/language/language-tour.md

```diff
@@ -308,13 +307,13 @@ You can also use it to create constant _values_,
 as well as to declare constructors that _create_ constant values.
 Any variable can have a constant value.
 
-<?code-excerpt "language-tour/const-vs-const/bin/main.dart"?>
+<?code-excerpt "lib/language_tour/variables.dart (const-vs-final)"?>
 {% prettify dart %}
 // Note: [] creates an empty list.
-// const [] creates an empty, immutable list (EIA).
-var foo = const []; // foo is currently an EIA.
-final bar = const []; // bar will always be an EIA.
-const baz = const []; // baz is a compile-time constant EIA.
+// const [] creates an empty, immutable list (EIL).
+var foo = const []; // foo is currently an EIL.
+final bar = const []; // bar will always be an EIL.
+const baz = const []; // baz is a compile-time constant EIL.
```

Comment:
"EIL" seemed like a better abbreviation for "empty, immutable list" than "EIA".

Reply:
Good catch. Looks like we did a global search-and-replace of array to list, and missed this consequence.